### ISSUE TITLE
margin/align name inconsistency

### ISSUE THREAD

**Issue by Xeverous** [cycfi/elements#145]
I just realized there is `align_left`, `align_right` etc but also `left_margin`, `right_margin`. Is this intended? IMO it would be good to have everything spelled one way.

**Comment by djowel** [comment]
Now that more users are already using elements, if we do this, it is time to use the [[deprecated]] feature. It is also probably time to have a stabilized version, on the way to v1.0 (perhaps v0.8). My idea is to move deprecated APIs in a separate directory, to keep the main API clean. Any API that is bound to change, that is already in the examples or already documented, should have a deprecation notice.

**Comment by Xeverous** [comment]
New thing: docs mention:

```cpp
// variant 1
vmargin({ top, bottom }, subject)

// variant 2
vmargin(top, bottom, subject)

// variant 3
top_bottom_margin({ top, bottom }, subject)

// variant 4
top_bottom_margin(top, bottom, subject)
```

But actually none of these functions exist. There are `xside_margin` and `yside_margin` instead.

**Comment by djowel** [comment]
Good catch! I'll look into it ASAP.

**Comment by djowel** [comment]
fixed in e2c2d22a6dcb2851a8b3cffa9e4157afb3b446ab
